test/client-test.js

```diff
@@ -838,5 +838,36 @@ var fs = require('fs'),
         });
       });
     });
+
+    describe('Method args sequence', function() {
+
+      // it('check if method required sequence args', function(done) {
+      //   soap.createClient(__dirname + '/wsdl/rpcexample.wsdl', meta.options, function(err, client) {
+      //     assert.ok(client);
+      //     assert.ok(client._isSequenceRequired(client.wsdl.definitions.bindings.RpcExample.methods.pullFile) === true);
+      //     done();
+      //   });
+      // });
+
+      it('check sort args on sequence required method', function(done) {
```

Comment:
how does this test guarantee the functionality of sorting the args in the correct "sequence" order?